samples/central-login/README.md

```diff
@@ -0,0 +1,39 @@
+# Central login
+
+In cases in which a centralized login application is desired, in contrast to an embedded login form, the SDK provides a method for leveraging an OAuth flow known as Authorization Code Flow (with PKCE). By using an option (login of "redirect") provided by `TokenManager` to request OAuth/OIDC tokens, when the user is not authenticated, the SDK will redirect the user to the configured login application with the ForgeRock platform. Once authenticated, the user will be redirected back to the original application to attain OAuth/OIDC tokens to complete the user flow.
+
+You can run this sample app with the `npm run start:central-login` command. Please [see the Getting Started instructions](#getting-started) for more details.
+
+### Getting Started
+
+1. Setup CORS support in an Access Management (AM) instance.
+
+   See [Enabling CORS Support](https://sdks.forgerock.com/js/01_prepare-am/#enabling-cors-support) in the Documentation.
+
+2. Create an authentication tree in AM.
+
+   See [Creating a User Authentication Tree](https://sdks.forgerock.com/js/01_prepare-am/#creating-a-user-authentication-tree) in the Documentation.
+
+3. Clone this repo:
+
+   ```
+   git clone https://github.com/ForgeRock/forgerock-javascript-sdk.git
+   ```
+
+4. In the root folder of the repo, use NPM to install dependencies:
+
+   ```
+   npm install
+   ```
+
+5. Open `samples/central-login/.env.example`. Copy the file in the same directory and name it `.env`. Fill in the values in this file with your values.
```

Comment:
Two things:

1. I'd like these instructions to align with what we instruct the reader to do in the official SDK docs, which would mean directly adding the values to the config object, rather than abstracting away the configuration
2. If we are going to leverage an environment file, can we use the root one, rather than a nested one?

Reply:
We can use the root .env file. I opted not to for one main reason.

The main reason was I didn't want to clutter the root .env file with env variables that would not be defined in the blog post(s).

We can totally use the root .env file and edit it like so

```bash
# reactjs & angular todo env variables
var1 ....



# central & embedded login apps
 var 1 .....
 ```

There are no conflicts in variables so, it's a reasonable thing to do.

Reply:
I think having a root .env common to the sample apps makes sense given that the vast majority of variables can probably be reused. Giving the ability to override at an app level should give flexibility where we may have developers running multiple sample apps at the same time